internal/pkg/cli/secret.go

```diff
@@ -0,0 +1,27 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"github.com/aws/copilot-cli/cmd/copilot/template"
+	"github.com/aws/copilot-cli/internal/pkg/cli/group"
+	"github.com/spf13/cobra"
+)
+
+// BuildSecretCmd is the top level command for secret.
+func BuildSecretCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "secret",
+		Short: `Commands for secrets.
+Copilot secrets can be versioned by existing Copilot environments in your application.`,
+	}
+
+	cmd.AddCommand(BuildSecretInitCmd())
+
+	cmd.SetUsageTemplate(template.Usage)
+	cmd.Annotations = map[string]string{
+		"group": group.Develop,
```

Comment:
I wonder if it'd be confusing if we put this under "Addons".   
I think of the "Addons" group as ways of augmenting services, my only concern is that users might expect a template as output. What do you think we rename this group to "Extend"?

```
$ copilot --help
Extend 🧸
    storage     Commands for working with storage and databases.
    secret      Commands for working with sensitive information.
```

Reply:
Yes I think `Extend` is a good characterization, and I agree that secret is better put here than `Develop` \